examples/templates/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Flagsmith Test</title>
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"
+    integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <p>
+    Hello <span id="identifier-name">{{ identifier ? identifier : 'World!'}}</span>,
+  </p>
+  {% if enabled %}
+ <button id="secret-button" style="color: {{ font_colour.colour }};">Secret Button</button>
+{% endif %}
+  <p>
+  <form id="form" method="get">
+    <h3>Identify as a user</h3>
+    <label for="identifier">Identifier: </label>
+    <input name="identifier" id="identifier" />{{ identifier ? identifier : ''}}
```

Comment:
I don't think we need the additional `{{ identifier ? identifier : '' }}` piece here?